jerry-core/api/jerry.c

```diff
@@ -50,29 +50,6 @@
 #include "js-parser.h"
 #include "re-compiler.h"
 
-JERRY_STATIC_ASSERT (sizeof (jerry_value_t) == sizeof (ecma_value_t),
```

Comment:
This checks should be kept. Otherwise I am ok with this patch, lets hear the others.